src/adapter.js

```diff
@@ -377,7 +444,9 @@ export function parse({minim, source, generateSourceMap}, done) {
         }
 
         if (methodValue.externalDocs) {
-          // TODO: [Annotation] Warn about unused external docs.
+          setupAnnotation(ANNOTATIONS.DATA_LOST,
+            `paths.${href}.${method}.externalDocs`,
+            'External documentation is not yet supported');
```

Comment:
Yeah, we could use a link, add a snippet describing the linked doc to the description/copy or we could just inline the content. I don't know which would be best (maybe an option makes sense?)